libs/perun/services/src/lib/auth.service.ts

```diff
@@ -72,13 +72,6 @@ export class AuthService {
     this.manager.events.addUserLoaded(user => {
       this.user = user;
     });
-
-    this.manager.events.addAccessTokenExpired(() => {
-      const config = getDefaultDialogConfig();
-      config.width = '450px';
-
-      this.dialog.open(SessionExpirationDialogComponent, config);
-    });
```

Comment:
check this. Was it there for some reason? Is it safe to delete it?

Reply:
I think so. Probably, it is used to display a dialog when the access token has expired. It shouldn't be probably removed.